docs/source/about/changelog.rst

```diff
@@ -23,6 +23,10 @@ more info, see the :ref:`Contributor Guide <Creating a Changelog Entry>`.
 Unreleased
 ----------
 
+**Changed**
+
+- :pull:`1050` - Warn when operating system has missing mime types, which can result in ``reactpy.run`` not working as expected.
```

Comment:
This will also warn and fix the issue if you're running in production.

```suggestion
- :pull:`1050` - Warn, and attempt to fix, when operating system has missing mime types.
```